os/board/rtl8721csm/src/component/common/bluetooth/realtek/sdk/example/da_ble_peripheral/ble_tizenrt_da_server.c

```diff
@@ -18,21 +18,11 @@
 #include "ble_tizenrt_scatternet_link_mgr.h"
 #include "ble_tizenrt_app.h"
 #include <os_mem.h>
-extern da_ble_result_type rtw_ble_combo_init(da_ble_client_init_parm* init_client, da_ble_server_init_parm* init_server);
-bool is_server_init = false;
 
+bool is_server_init = false;
```

Comment:
If `is_server_init` is used only within this file, please add static.

Reply:
It also used in ble_tizenrt_da_combo.c.